src/monmove.cpp

```diff
@@ -824,6 +840,15 @@ int monster::attack_at(int x, int y) {
 
 int monster::move_to(int x, int y, bool force)
 {
+    //Allows climbing monsters to move on terrain with movecost <= 0
+    if (g->m.has_flag_ter_or_furn("CLIMBABLE", x, y) && has_flag(MF_CLIMBS) && g->npc_at(x, y) == -1 && g->mon_at(x, y) == -1) {
+            if (g->u.sees( *this )){
+                    add_msg(_("The %s climbs over the %1$s."), name().c_str(),
+                            g->m.has_flag_ter_or_furn("CLIMBABLE", x, y) ? g->m.tername(x, y).c_str() : g->m.furnname(x, y).c_str());
```

Comment:
When building without localization, the numbering is stripped away (for compatibility with some broken platform or other). For this reason, translatable strings must always be written as if the numbers will be sequential, and must have exactly as many values as formatting markers. Translated strings have more leeway -- building with gettext on the aforementioned broken platform is not supported -- so their numbers need not be sequential.

Also, to reiterate (and hopefully clarify), formatting indices start at 1, not 0, therefore `"the %1$s"` is a reference to the first value given to `add_msg`, not the second. The intent here seems to be either `"The %1$s climbs over the %2$s."`, or just simply `"The %s climbs over the %s."`. Personally, I'd do as @BevapDin suggests and go with the latter.